README.md

```diff
@@ -7,8 +7,12 @@
   <br />
   <a title="Learn more about Separators" href="http://github.com/alefragnani/vscode-separators"><img src="https://raw.githubusercontent.com/alefragnani/vscode-separators/master/images/vscode-separators-logo-readme.png" alt="Separators Logo" width="70%" /></a>
 </p>
+## Seperators
 
-# What's new in Separators 2.2
+This extension adds subtle colored line delimiters between different sections of code to ease visual navigation.
```

Comment:
The current paragraph is:
> It improves the readability of your source code, by drawing lines on top of each symbol.

Merging with your suggestion, I think it could become:

_“It improves the readability of your source code, adding subtle colored lines on top of each symbol.”_

Plus, some of the keywords you provided in #44 could be added to `package.json`, which would increase the marketplace search engine.